akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/HttpChallenge.scala

```diff
@@ -13,7 +13,8 @@ final case class HttpChallenge(scheme: String, realm: String,
                                params: Map[String, String] = Map.empty) extends jm.headers.HttpChallenge with ValueRenderable {
```

Comment:
Ouch, default parameters are a killer for binary compat, too bad we had one here :-/ 
Yes, @2beaucoup's suggestion is what we'll need to go with here